src/collectors/windows.plugin/integrations/ms_sql_server.md

```diff
@@ -88,7 +88,11 @@ Metrics:
 | mssql.instance_bufman_iops | read, written | pages/s |
 | mssql.instance_bufman_checkpoint_pages | log | pages/s |
 | mssql.instance_bufman_page_life_expectancy | life_expectancy | seconds |
+| mssql.instance_bufman_lazy_write | lazy_write | Lazy writes/sec |
+| mssql.instance_bufman_page_lookups | page_lookups | Page lookups/sec |
```

Comment:
> Lazy writes/sec
> Page lookups/sec

Did you really make these units, @thiagoftsm? They are wrong. If unsure, please ask me or @ralphm next time.

Reply:
These names are defined by [Microsfot](https://github.com/netdata/netdata/blob/8f20f6278058708a7b83b87512fd28f1dc8d2982/src/collectors/windows.plugin/perflib-mssql-queries.h#L22).

While there's no technical issue with renaming them, would this change actually benefit users?

Reply:
Yes, it will make a difference. I recommend reviewing existing Netdata charts and metrics to understand our conventions. While we don't have a formal guide yet, the patterns are consistent and straightforward to follow. For example, we use `/s` rather than `/sec`, lowercase for units, and typically single words (e.g., `writes/s` instead of `Lazy writes/sec`). Maintaining these standards helps ensure consistency across the project and benefits everyone working with the codebase.

Reply:
PR https://github.com/netdata/netdata/pull/20864 brings your suggestions @ilyam8. Thank you!